.github/workflows/deploy-admin-app-s3.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       - name: Build
         working-directory: ./admin-app
         run: yarn build
+      - name: Create env file
+        working-directory: ./admin-app
+        run: echo "$VITE_HEX_ADMIN_API_URL" > .env
```

Comment:
You're mixing up the definition of **one** env var with the definition of a whole `.env` file.

I suggest stg like `echo "$ADMIN_ENV_VARS" > .env`
with `ADMIN_ENV_VARS` containing something like:

```
VITE_HEX_ADMIN_API_URL=https://d1iqp7uyjrmy2a.cloudfront.net/api/admin
```

Reply:
Oupsy doupsy !